src/silverlabnwb/generate_extended_schema.py

```diff
@@ -59,11 +61,27 @@ def generate_extended_schema():
                                             neurodata_type_def='SilverLabMetaData',
                                             neurodata_type_inc='LabMetaData',
                                             )
+
+    # dimensions ordered as t, x, y [, z], like the TimeSeries data itself
+    silverlab_pixel_time_offset_data = NWBDatasetSpec(doc='A datastructure to hold time offsets for pixels. The'
+                                                          'time offsets are the acquisition time of each pixel '
+                                                          'relative to the starting time/timestamp of a '
+                                                          'ROISeriesWithPixelTimeOffsets.',
```

Comment:
Is it relative to the starting time (= time of the cycle?) or to the start of the trial?